easybuild/easyconfigs/p/PyTorch-extensions/PyTorch-extensions-2019.01-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'PythonBundle'
+
+name = 'PyTorch-extensions'
+version = '2019.01'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/rusty1s'
+description = """This is a bundle of extension libraries for PyTorch written by Matthias Fey."""
```

Comment:
homepage/description should be changed too if name is changed to `PyTorch-Geometric`

Reply:
In #9976 you left it this way. Should I still change it?

Reply:
Ugh, no, I should have changed that too... Will fix.

Reply:
fixed in #10020